code/modules/organs/external/wounds/wound.dm

```diff
@@ -203,7 +206,7 @@
 			return FALSE
 	if(bandaged || clamped)
```

Comment:
could maybe replace the bandaged check here with a !is_open() check, removing the one in the final return